rebar.config

```diff
@@ -38,11 +38,9 @@
               ]}.
 
 {edoc_opts, [ {doclet, edown_doclet}
-            , {dialyzer_specs, all}
-            , {report_missing_type, true}
-            , {report_type_mismatch, true}
-            , {pretty_print, erl_pp}
+            , {report_missing_types, true}
             , {preprocess, true}
+            , {app_default, "http://www.erlang.org/doc/man"} % https not handled correctly
```

Comment:
Https not handled correctly???

It's not very nice to generate http links...

Reply:
Yes, annoying..
I found a fix for it in edown, lets see if it's good enough. Will prepare a PR.

Reply:
A fix for this is now on edown's master, yey